### DISCUSSION TITLE
[Bug] Reinstall with saved data : still trying to install

### DISCUSSION THREAD

**Discussion by Draky50110** [FreshRSS/FreshRSS#6745] [Q&A] [answered]
### Describe the bug

Hello.

I wanted to "reinstall" using fresh ZIP file from GitHub.
I uncompressed it than put back data/config.php + data/user dir and when accessing URL, it stills stry to install itself, even if config.php in data dir is ok with MySQL credentials.

Did I miss something (file ?) to save/put back ?

Thanks

### To Reproduce

Uncompressed ZIP
Get back data/config.php + data/users dir

### FreshRSS version

1.24.2

### Environment information

Linux

**Comment by math-GH** [comment]
Is your reinstall with the same URL? Check `base_url` in `config.php`

**Comment by Alkarex** [comment, marked as answer]
Try to create a file `data/applied_migrations.txt`:

```sh
cd ./FreshRSS/
touch data/applied_migrations.txt
```

**Reply by Draky50110** [reply]
That worked, thanks !
